golem-worker-service/src/api/openapi/converter.rs

```diff
@@ -0,0 +1,1019 @@
+                    r#in: ParameterLocation::Query,
+                    schema: Schema::Integer { format: Some("uint64".to_string()) },
+                    style: Some("form".to_string()),
+                    explode: Some(true),
+                    required: Some(false),
+                    description: None,
+                }
+            );
+             params.push(
+                Parameter {
+                    name: "count".to_string(),
+                    r#in: ParameterLocation::Query,
+                     schema: Schema::Integer { format: Some("uint64".to_string()) },
+                    style: Some("form".to_string()),
+                    explode: Some(true),
+                    required: Some(true),
+                    description: None,
+                }
+            );
+             params.push(
+                Parameter {
+                    name: "cursor".to_string(),
+                    r#in: ParameterLocation::Query,
+                     schema: Schema::Ref {
+                        reference: "#/components/schemas/OplogCursor".to_string()
+                     },
+                    style: Some("form".to_string()),
+                    explode: Some(true),
+                    required: Some(false),
+                    description: None,
+                }
+            );
+              params.push(
+                Parameter {
+                    name: "query".to_string(),
+                    r#in: ParameterLocation::Query,
+                    schema: Schema::String { format: None, enum_values: None },
+                    style: Some("form".to_string()),
+                    explode: Some(true),
+                    required: Some(false),
+                    description: None,
+                }
+            );
+        }
+         if route.path.contains("/download") {
+              params.push(
+                Parameter {
+                    name: "version".to_string(),
+                    r#in: ParameterLocation::Query,
+                     schema: Schema::Integer { format: Some("uint64".to_string()) },
+                    style: Some("form".to_string()),
+                    explode: Some(true),
+                    required: Some(false),
+                    description: None,
+                }
+            );
+         }
+        if route.path.contains("/components") && route.method == HttpMethod::Get {
+              params.push(
+                Parameter {
+                    name: "component-name".to_string(),
+                    r#in: ParameterLocation::Query,
+                     schema: Schema::String { format: None, enum_values: None },
+                    style: Some("form".to_string()),
+                    explode: Some(true),
+                    required: Some(false),
+                    description: None,
+                }
+            );
+        }
+        if route.path.contains("/api/definitions") && route.method == HttpMethod::Get {
```

Comment:
Again, all this special-case logic doesn't begin here. We want to generate OpenAPI for user-defined APIs (which are defined with file server, swagger, and default / worker binding types), not for Golem REST APIs.